aws-ts-pulumi-webhooks/index.ts

```diff
@@ -53,19 +55,23 @@ webhookHandler.get("/", async (_, res) => {
 });
 
 webhookHandler.post("/", logRequest, authenticateRequest, async (req, res) => {
-    const webhookKind = req.headers["pulumi-webhook-kind"];
+    const webhookKind = req.headers["pulumi-webhook-kind"].toString();
     const payload = <string>req.body.toString();
     const prettyPrintedPayload = JSON.stringify(JSON.parse(payload), null, 2);
+    const fallbackText = `Pulumi Service Webhook (\`${webhookKind}\`)\n` + "```\n" + prettyPrintedPayload + "```\n";
```

Comment:
Nit: Since this is only used inside of `messageArgs`, can we put its declaration to right above it?

Reply:
> And my understanding is that TypeScript will then figure out the right overload to call based on that string's value.

Indeed, I was not aware of this fanciness! After a good bit of fiddling, though, I couldn't quite bring it together, so I've pulled the formatters into functions as you suggested, and that does feel tidier, if not quite ✨.